core/src/main/scala/org/apache/spark/api/python/WriteInputFormatTestDataGenerator.scala

```diff
@@ -56,33 +57,39 @@ case class TestWritable(var str: String, var int: Int, var double: Double) exten
   }
 }
 
+@deprecated("This class will be move to `test`.", "3.5.2")
 private[python] class TestInputKeyConverter extends Converter[Any, Any] {
   override def convert(obj: Any): Char = {
     obj.asInstanceOf[IntWritable].get().toChar
   }
 }
 
+@deprecated("This class will be move to `test`.", "3.5.2")
 private[python] class TestInputValueConverter extends Converter[Any, Any] {
   override def convert(obj: Any): ju.List[Double] = {
     val m = obj.asInstanceOf[MapWritable]
     m.keySet.asScala.map(_.asInstanceOf[DoubleWritable].get()).toSeq.asJava
   }
 }
 
+@deprecated("This class will be move to `test`.", "3.5.2")
 private[python] class TestOutputKeyConverter extends Converter[Any, Any] {
   override def convert(obj: Any): Text = {
     new Text(obj.asInstanceOf[Int].toString)
   }
 }
 
+@deprecated("This class will be move to `test`.", "3.5.2")
 private[python] class TestOutputValueConverter extends Converter[Any, Any] {
```

Comment:
`private[python]`  probably doesn't need to add `@deprecated`, right?

Reply:
Yeah, according to this principle, we only need to focus on whether the method is `public`, rather than `internal usage`